backend_new/test/unit/services/jurisdiction.service.spec.ts

```diff
@@ -0,0 +1,371 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { PrismaService } from '../../../src/services/prisma.service';
+import { JurisdictionService } from '../../../src/services/jurisdiction.service';
+import { JurisdictionCreate } from '../../../src/dtos/jurisdictions/jurisdiction-create.dto';
+import { JurisdictionUpdate } from '../../../src/dtos/jurisdictions/jurisdiction-update.dto';
+import { LanguagesEnum } from '@prisma/client';
+import { randomUUID } from 'crypto';
+
+describe('Testing jurisdiction service', () => {
+  let service: JurisdictionService;
+  let prisma: PrismaService;
+
+  const mockJurisdiction = (position: number, date: Date) => {
+    return {
+      id: randomUUID(),
+      createdAt: date,
+      updatedAt: date,
+      name: `jurisdiction ${position}`,
+      notificationsSignUpUrl: `notificationsSignUpUrl: ${position}`,
+      languages: [LanguagesEnum.en],
+      partnerTerms: `partnerTerms: ${position}`,
+      publicUrl: `publicUrl: ${position}`,
+      emailFromAddress: `emailFromAddress: ${position}`,
+      rentalAssistanceDefault: `rentalAssistanceDefault: ${position}`,
+      enablePartnerSettings: true,
+      enableAccessibilityFeatures: true,
+      enableUtilitiesIncluded: true,
+    };
+  };
+
+  const mockJurisdictionSet = (numberToCreate: number, date: Date) => {
+    const toReturn = [];
+    for (let i = 0; i < numberToCreate; i++) {
+      toReturn.push(mockJurisdiction(i, date));
+    }
+    return toReturn;
+  };
+
+  beforeAll(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [JurisdictionService, PrismaService],
+    }).compile();
+
+    service = module.get<JurisdictionService>(JurisdictionService);
+    prisma = module.get<PrismaService>(PrismaService);
+  });
+
+  it('testing list()', async () => {
+    const date = new Date();
+    const mockedValue = mockJurisdictionSet(3, date);
+    prisma.jurisdictions.findMany = jest.fn().mockResolvedValue(mockedValue);
+
+    expect(await service.list()).toEqual([
+      {
+        id: mockedValue[0].id,
```

Comment:
Similar feedback from other PRs that I think a helper would be nice for these objects

Reply:
I went ahead and updated these its not so much a helper but since we shouldn't be modifying the results from what prisma returns the mock value should match with the returned value